distribution/docs/src/main/resources/_contents/_securing/authentication-components-contents.adoc

```diff
@@ -0,0 +1,114 @@
+
+==== Authentication Components
+
+It is assumed that the three authentication components identified below are installed on three separate servers.
```

Comment:
Actually, much of this is redundant with other sections.
Made some major revisions to this section. 
- rewrote intro paragraph to clarify authentication components. 
- Replace LDAP section with A) link to duplicated embedded LDAP instructions. B) added note that should use external LDAP.
- dropped verbiage about STS-only installation.